script.js

```diff
@@ -0,0 +1,116 @@
+//interactive navigation menu
+document.addEventListener('scroll', onScroll);
+
+function onScroll(event) {
+  const curPos = window.scrollY;
+  const ancher_block = document.querySelectorAll('#body>div, section');
+  const links = document.querySelectorAll('#menu a');
+  const linksBurger = document.querySelectorAll('#menu-burger a');
+  ancher_block.forEach((el) => {
```

Comment:
Better camelCase naming for JS constants.

Reply:
DONE!